Automattic/woocommerce-payments
# Change the payload sent to unpause a subscription

Fixes #2829

#### Changes proposed

A small change to the unpause subscription payload to remove `cancel_at_period_end` as it's not necessary as per the Stripe docs - https://stripe.com/docs/billing/subscriptions/pause#unpausing

Their PHP code reference looks like this:

```php
// Set your secret key. Remember to switch to your live secret key in production.
// See your keys here: https://dashboard.stripe.com/apikeys
\Stripe\Stripe::setApiKey('sk_test_51HHncdJ5cIRIG92xN0v0tyniYQTOdA89szB1YcrC74dTHnbMLZPVXaYxiymtMrFHxAb6xz0QEh1D9qu33omtbgug00REkyBJyU');

\Stripe\Subscription::update(
  'sub_GTbTiykEwMRog0',
  [
    'pause_collection' => '',
  ]
);
```

#### Testing instructions

This should be tested with this server PR - 1154-gh-Automattic/woocommerce-payments-server

From the WCPay client:

1. If you have the subscriptions plugin activated, deactivate this to use WCPay built-in subscriptions.
2. Purchase a new subscription (you may need to create a new WCPay subscription product if you don't have one)
2. On checkout complete, confirm the subscription is activated on your connected Stripe account
3. Suspend the subscription Woo
4. Confirm the subscription is paused in the Stripe dashboard
5. Reactivate the subscription in Woo.
6. Confirm the subscription is resumed (active) in the Stripe dashboard.

-------------------


**Post merge**
